fedmsg_meta_fedora_infrastructure/fedimg.py

```diff
@@ -66,9 +66,37 @@ def subtitle(self, msg, **config):
             return tmpl.format(image_name=name, dest=dest, ami=ami,
                                virt=virt, vol=vol)
 
+        if 'image.publish' in msg['topic']:
+            tmpl = ''
+            ami = extra.get('id')
```

Comment:
Maybe assign value to ami, virt and vol outside if statement and reuse them? For example: after https://github.com/sayanchowdhury/fedmsg_meta_fedora_infrastructure/blob/ab094b1b93b7d8bc215a0d21e003c6f9e1b47d37/fedmsg_meta_fedora_infrastructure/fedimg.py#L36 when extra is not empty.

Reply:
+1